docs/api.md

```diff
@@ -83,6 +83,9 @@ Keys used to open the command bar.
 Default admin level that will be given to players
 when the join.
 
+#### `Configuration.DefaultServerLockAdminLevel: number`
+Default admin level to use in "slock" when a AdminLevel is not provided.
```

Comment:
Changing this was missed. There is now `CommandConfigurations` that contains this.